Vagrantfile

```diff
@@ -23,23 +23,36 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     c.vm.hostname = "centos-build"
     c.vm.box = "opscode_centos-6.5"
     c.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-6.5_chef-provisionerless.box"
-  
+
     c.vm.provision "shell", inline: "sudo yum install -y git rpm-build python-configobj"
-    c.vm.provision "shell", inline: "cp -rf /vagrant /tmp/Diamond"  
+    c.vm.provision "shell", inline: "cp -rf /vagrant /tmp/Diamond"
     c.vm.provision "shell", inline: "cd /tmp/Diamond && make rpm"
     c.vm.provision "shell", inline: "cd /tmp/Diamond/dist && sudo bash -c 'for f in *.src.rpm; do mv $f `basename $f .src.rpm`.el6.src.rpm; done;'"
     c.vm.provision "shell", inline: "cd /tmp/Diamond/dist && sudo bash -c 'for f in *.noarch.rpm; do mv $f `basename $f .noarch.rpm`.el6.noarch.rpm; done;'"
     c.vm.provision "shell", inline: "mkdir -p /vagrant/dist/el6 && (cp -f /tmp/Diamond/dist/* /vagrant/dist/el6/ || grep -v 'cannot stat')"
   end
 
+  config.vm.define "centos7-build" do |c|
+    c.vm.hostname = "centos-build"
+    c.vm.box = "opscode_centos-7.2"
+    c.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-7.2_chef-provisionerless.box"
+
+    c.vm.provision "shell", inline: "sudo yum install -y git rpm-build python-configobj"
+    c.vm.provision "shell", inline: "cp -rf /vagrant /tmp/Diamond"
+    c.vm.provision "shell", inline: "cd /tmp/Diamond && make rpm"
+    c.vm.provision "shell", inline: "cd /tmp/Diamond/dist && sudo bash -c 'for f in *.src.rpm; do mv $f `basename $f .src.rpm`.el7.src.rpm; done;'"
+    c.vm.provision "shell", inline: "cd /tmp/Diamond/dist && sudo bash -c 'for f in *.noarch.rpm; do mv $f `basename $f .noarch.rpm`.el7.noarch.rpm; done;'"
+    c.vm.provision "shell", inline: "mkdir -p /vagrant/dist/el6 && (cp -f /tmp/Diamond/dist/* /vagrant/dist/el6/ || grep -v 'cannot stat')"
+  end
+
   config.vm.define "ubuntu-build" do |c|
     c.vm.hostname = "ubuntu-build"
     c.vm.box = "opscode_ubuntu-12.04"
     c.vm.box_url = "http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-12.04_chef-provisionerless.box"
-  
-    c.vm.provision "shell", inline: "sudo apt-get update"  
+
+    c.vm.provision "shell", inline: "sudo apt-get update"
```

Comment:
Is it even a good idea to update a system's apt repo blindly? Seems like this might change more about the vagrant box than what someone may have wanted.